src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -32,6 +32,12 @@
 import tempfile
 import requests
 
+from .aaz.latest.network.express_route import Create as _ExpressRouteCreate, Update as _ExpressRouteUpdate
+from .aaz.latest.network.express_route.gateway.connection import Create as _ExpressRouteConnectionCreate, \
+    Update as _ExpressRouteConnectionUpdate
+from .aaz.latest.network.express_route.peering import Create as _ExpressRoutePeeringCreate, \
+    Update as _ExpressRoutePeeringUpdate
+
```

Comment:
Please move this code block to "from .aaz.latest.network.application_gateway._update import Update as _ApplicationGatewayUpdate" and keep the lexicographical order.